Changelog.md

```diff
@@ -2,12 +2,26 @@ Changelog
 =========
 
 
-Current master
+Master: Version table
+-------------
+
+| Swift version | RxSwift version | Action version |
+| ------------- | --------------- | -------------- |
+| Swift 3.0     | v3.2.*             | v2.2.0            |
+| Swift 3.2     | v3.6.*             | v3.2.0            |
+| **Swift 4**   | **v4.0.0**      | **v3.4.0**     |
+| Swift 4.1    | **v4.2.0**      | **v3.6.0**     |
+| Swift 4.2    | **v4.3.1**      |   **v4.0**     |
```

Comment:
Can we align the markdown? I know its rendered right but as a person who likes to read raw markdown it bugs me :P

Reply:
Yeah, I will align them, just want to keep the PR is clean by keeping the original code.